libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,15 +35,9 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
+    this.subscription$ = this.store.select(getAllBooks).subscribe(books => {
```

Comment:
Great that the subscription reference is held and unsubscribed from on destroy!

Note that we generally don't use the `$` suffix for subscriptions, only for observables.

Reply:
Oh more thing, if you use the `async` pipe in your template then you can avoid managing subscriptions altogether!

e.g.
```
export class BookSearchComponent {
  books$ = this.store.select(getAllBooks);
  // ...
}
```

Then in your template:

```
*ngFor="let book of books$ | async"
```

Reply:
@jaysoo: This has been taken care. Please take a look.